modules/Crowd Assessment/moduleFunctions.php

```diff
@@ -26,38 +26,38 @@ function getLessons($guid, $connection2, $and = '')
 
     $fields = 'gibbonPlannerEntryID, gibbonUnitID, gibbonCourse.nameShort AS course, gibbonCourseClass.nameShort AS class, gibbonPlannerEntry.name, timeStart, timeEnd, viewableStudents, viewableParents, homework, homeworkDetails, date, gibbonPlannerEntry.gibbonCourseClassID, homeworkCrowdAssessOtherTeachersRead, homeworkCrowdAssessClassmatesRead, homeworkCrowdAssessOtherStudentsRead, homeworkCrowdAssessSubmitterParentsRead, homeworkCrowdAssessClassmatesParentsRead, homeworkCrowdAssessOtherParentsRead';
     //Get my classes (student, teacher, classmates)
-    $data = array('today1' => $today, 'gibbonPersonID1' => $_SESSION[$guid]['gibbonPersonID'], 'now1' => $now, 'gibbonSchoolYearID1' => $_SESSION[$guid]['gibbonSchoolYearID']);
+    $data = array('today1' => $today, 'gibbonPersonID1' => $session->get('gibbonPersonID'), 'now1' => $now, 'gibbonSchoolYearID1' => $session->get('gibbonSchoolYearID'));
```

Comment:
We can't make changes to the moduleFunctions.php script at this time. Can you revert these changes please? Thanks!